0_RootFS/PlatformSupport/build_tarballs.jl

```diff
@@ -122,8 +127,7 @@ case "${COMPILER_TARGET}" in
                  -DCMAKE_INSTALL_PREFIX="${sysroot}/usr" \
                  -DCMAKE_CROSSCOMPILING=True \
                  -DLLVM_HOST_TRIPLE=${COMPILER_TARGET} \
-                 -DDARWIN_macosx_CACHED_SYSROOT:STRING="${sysroot}" \
-                 -DDARWIN_osx_ARCHS=x86_64
```

Comment:
Elliot verified that this flag wasn't been used and the resulting tarballs had the same checksum